apps/airdrop/src/App/Claim/ClaimConfirmation/ClaimConfirmation.tsx

```diff
@@ -114,6 +115,14 @@ export const ClaimConfirmation: React.FC = () => {
   const [tsSignature, setTsSignature] = useState<string>("");
   const [isToggleChecked, setIsToggleChecked] = useState(false);
 
+  const isTrustedSetup = claimState?.type === "ts";
+  const isClaimSectionActive = !isTrustedSetup || !!tsSignature;
+  const airdropAddressValid = bech32mValidation("tnam", airdropAddress);
+
+  const isValidForm = Boolean(
+    airdropAddressValid && airdropPubKey && isToggleChecked
```

Comment:
If the TS signature is removed after the form is completed, `isValidForm` will still be true and the big nuclear claim button will still be shown.

Reply:
Also the same behaviour in the case where after the form is completed, `airdropPubKey` is changed from something valid to something invalid, because this checks whether `airdropPubKey` exists instead of whether it is valid.